src/main/java/http/HttpMethod.java

```diff
@@ -24,15 +24,14 @@ private static boolean isHttpMethod(String methodInput) {
         return HTTP_METHODS.contains(methodInput);
     }
 
+    public static boolean isGET(String httpMethod) {
+        return GET.name().equals(httpMethod);
```

Comment:
enum 으로 선언된 HttpMethod 를 갖는 RequestLine 클래스의 httpMethod 는 타입이 HttpMethod 인데, 
String 으로 변환 후 비교할 필요가 없어 보이네요~
변환하지 않는다면 static 메서드가 아니여도 될것 같아요. 😄 
```suggestion
        return this == GET;
```